<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>MaestroControl</class>
 <widget class="QWidget" name="MaestroControl">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>405</width>
    <height>280</height>
   </rect>
  </property>
  <property name="sizePolicy">
   <sizepolicy hsizetype="Ignored" vsizetype="Ignored">
    <horstretch>0</horstretch>
    <verstretch>0</verstretch>
   </sizepolicy>
  </property>
  <property name="minimumSize">
   <size>
    <width>405</width>
    <height>0</height>
   </size>
  </property>
  <property name="windowTitle">
   <string>Form</string>
  </property>
  <layout class="QGridLayout" name="gridLayout_2">
   <item row="0" column="2">
    <layout class="QHBoxLayout" name="canvasControlsLayout"/>
   </item>
   <item row="0" column="1">
    <layout class="QHBoxLayout" name="extraControlsLayout"/>
   </item>
   <item row="0" column="0">
    <layout class="QFormLayout" name="controlLayout">
     <item row="0" column="0">
      <widget class="QLabel" name="sectionSelectLabel">
       <property name="text">
        <string>Active Section</string>
       </property>
      </widget>
     </item>
     <item row="0" column="1">
      <layout class="QHBoxLayout" name="sectionSelectLayout">
       <item>
        <widget class="QComboBox" name="sectionComboBox"/>
       </item>
       <item>
        <widget class="QComboBox" name="canvasComboBox"/>
       </item>
      </layout>
     </item>
     <item row="1" column="0">
      <widget class="QLabel" name="mixModeLabel">
       <property name="text">
        <string>Mix Mode / Alpha</string>
       </property>
      </widget>
     </item>
     <item row="1" column="1">
      <layout class="QHBoxLayout" name="overlayLayout">
       <item>
        <widget class="QComboBox" name="mix_modeComboBox"/>
       </item>
       <item>
        <widget class="QSpinBox" name="alphaSpinBox">
         <property name="maximum">
          <number>255</number>
         </property>
         <property name="singleStep">
          <number>5</number>
         </property>
        </widget>
       </item>
      </layout>
     </item>
     <item row="2" column="0">
      <widget class="QLabel" name="gridSizeLabel">
       <property name="text">
        <string>Grid Size</string>
       </property>
      </widget>
     </item>
     <item row="2" column="1">
      <layout class="QHBoxLayout" name="sectionOptionsLayout">
       <item>
        <widget class="QSpinBox" name="rowsSpinBox">
         <property name="frame">
          <bool>true</bool>
         </property>
         <property name="buttonSymbols">
          <enum>QAbstractSpinBox::PlusMinus</enum>
         </property>
         <property name="minimum">
          <number>1</number>
         </property>
         <property name="maximum">
          <number>65535</number>
         </property>
         <property name="value">
          <number>10</number>
         </property>
        </widget>
       </item>
       <item>
        <widget class="QSpinBox" name="columnsSpinBox">
         <property name="buttonSymbols">
          <enum>QAbstractSpinBox::PlusMinus</enum>
         </property>
         <property name="minimum">
          <number>1</number>
         </property>
         <property name="maximum">
          <number>65535</number>
         </property>
         <property name="value">
          <number>10</number>
         </property>
        </widget>
       </item>
      </layout>
     </item>
     <item row="3" column="0">
      <widget class="QLabel" name="animationLabel">
       <property name="minimumSize">
        <size>
         <width>100</width>
         <height>0</height>
        </size>
       </property>
       <property name="text">
        <string>Animation</string>
       </property>
      </widget>
     </item>
     <item row="3" column="1">
      <layout class="QHBoxLayout" name="animationLayout">
       <item>
        <widget class="QComboBox" name="animationComboBox">
         <property name="minimumSize">
          <size>
           <width>150</width>
           <height>0</height>
          </size>
         </property>
        </widget>
       </item>
       <item>
        <widget class="QComboBox" name="orientationComboBox"/>
       </item>
      </layout>
     </item>
     <item row="5" column="1">
      <layout class="QHBoxLayout" name="animationOptionsLayout">
       <item>
        <widget class="QCheckBox" name="reverse_animationCheckBox">
         <property name="text">
          <string>Reverse</string>
         </property>
        </widget>
       </item>
       <item>
        <widget class="QCheckBox" name="fadeCheckBox">
         <property name="text">
          <string>Fade</string>
         </property>
         <property name="checked">
          <bool>true</bool>
         </property>
        </widget>
       </item>
      </layout>
     </item>
     <item row="7" column="0">
      <widget class="QLabel" name="colorLabel">
       <property name="minimumSize">
        <size>
         <width>100</width>
         <height>0</height>
        </size>
       </property>
       <property name="text">
        <string>Color Palette</string>
       </property>
      </widget>
     </item>
     <item row="7" column="1">
      <layout class="QHBoxLayout" name="colorSchemeLayout">
       <item>
        <widget class="QComboBox" name="colorComboBox">
         <property name="minimumSize">
          <size>
           <width>150</width>
           <height>0</height>
          </size>
         </property>
         <property name="currentText">
          <string/>
         </property>
         <property name="sizeAdjustPolicy">
          <enum>QComboBox::AdjustToContents</enum>
         </property>
        </widget>
       </item>
       <item>
        <widget class="QPushButton" name="paletteControlButton">
         <property name="text">
          <string>Edit Palettes</string>
         </property>
        </widget>
       </item>
      </layout>
     </item>
     <item row="9" column="0">
      <widget class="QLabel" name="cycleLabel">
       <property name="minimumSize">
        <size>
         <width>100</width>
         <height>0</height>
        </size>
       </property>
       <property name="text">
        <string>Speed</string>
       </property>
      </widget>
     </item>
     <item row="9" column="1">
      <layout class="QHBoxLayout" name="speedLayout">
       <item>
        <widget class="QSlider" name="cycleSlider">
         <property name="minimum">
          <number>10</number>
         </property>
         <property name="maximum">
          <number>3000</number>
         </property>
         <property name="singleStep">
          <number>10</number>
         </property>
         <property name="value">
          <number>1000</number>
         </property>
         <property name="orientation">
          <enum>Qt::Horizontal</enum>
         </property>
         <property name="invertedAppearance">
          <bool>false</bool>
         </property>
         <property name="invertedControls">
          <bool>false</bool>
         </property>
         <property name="tickPosition">
          <enum>QSlider::TicksBelow</enum>
         </property>
         <property name="tickInterval">
          <number>200</number>
         </property>
        </widget>
       </item>
       <item>
        <widget class="QSpinBox" name="cycleSpinBox">
         <property name="maximum">
          <number>3000</number>
         </property>
         <property name="singleStep">
          <number>10</number>
         </property>
         <property name="value">
          <number>1000</number>
         </property>
        </widget>
       </item>
      </layout>
     </item>
     <item row="10" column="1">
      <widget class="QLabel" name="lastActionLabel">
       <property name="text">
        <string>Last Action</string>
       </property>
      </widget>
     </item>
    </layout>
   </item>
  </layout>
 </widget>
 <resources/>
 <connections/>
</ui>
